### PR TITLE
PAYARA-1385-Make-the-Request-Tracing-Healthcheck-Services-create-empty-Notifiers-for-each-Notifier

### PR DESCRIPTION
**Pull request by mulderbaba**

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test please

File: nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java [hunk 1]
```diff
@@ -123,12 +126,33 @@ public Object run(final NotificationServiceConfiguration notificationServiceConf
                         applyValues(c);
                         notificationServiceConfigurationProxy.getNotifierConfigurationList().add(c);
                         actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
+
+                        ParameterMap params = new ParameterMap();
+                        params.add("enabled", Boolean.FALSE.toString());
```
Comment (Pandrex247) [inline nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java]: Can this be set to the value of the _enabled_ parameter please? One of the goals of this is to skip out the step of having to enable the notifiers for the healthcheck and request tracing services.

Can you also add in the _dynamic_ parameter as well please?

**Comment by mulderbaba** [comment]
jenkins test please

File: nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java (hunk 1 above)
Comment (mulderbaba) [inline nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java]: done

File: nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java [hunk 2]
```diff
@@ -123,12 +126,34 @@ public Object run(final NotificationServiceConfiguration notificationServiceConf
                         applyValues(c);
                         notificationServiceConfigurationProxy.getNotifierConfigurationList().add(c);
                         actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
+
+                        ParameterMap params = new ParameterMap();
+                        params.add("enabled", Boolean.TRUE.toString());
+                        params.add("dynamic", Boolean.TRUE.toString());
```
Comment (Pandrex247) [inline nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java]: Sorry, don't think I explained it properly.
I'd like these to be set to the parameters passed in from the command line, so something like:

```
params.add("enabled", enabled.toString());
if (dynamic != null) {
    params.add("dynamic", dynamic.toString());
}

**Pull request merged**